lib/ruby_lsp/addon/process_server.rb

```diff
@@ -0,0 +1,50 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "sorbet-runtime"
```

Comment:
We had some issues in the past when requiring `sorbet-runtime` within the Rails side. I can't recall the exact cause, but we might need to be wary of that or do some extra checking.